.env.sample

```diff
@@ -0,0 +1,2 @@
+VITE_NOTION_DATABASE_ID=""
```

Comment:
it is not a build-environment property, but a user configuration that should be passed with the initialization of the module

Reply:
Correct, we should not store secrets at the repo. They will be passed from a user on initialization.

There is no runtime environment since it is not a nodejs. So there is only a build-time environment that is not related to the notion credentials. 

I see no reason to implement a logic for working with the .env file to show the example config and remove this at the next PRs. You can simply show an example in the README and JSdoc.